ci-operator/step-registry/openstack/conf/generateconfig/openstack-conf-generateconfig-ref.yaml

```diff
@@ -8,6 +8,7 @@ ref:
       memory: 100Mi
   env:
     - name: OS_CLOUD
+      default: "openstack"
```

Comment:
again, don't think we actually need this. make validate-step-registry will catch it.